target/linux/ipq40xx/image/Makefile

```diff
@@ -241,6 +241,21 @@ define Device/engenius_eap1300
 endef
 TARGET_DEVICES += engenius_eap1300
 
+define Device/engenius_emd1
+	$(call Device/FitImage)
```

Comment:
Does the bootloader support LZMA packed kernels? If so, it's preferred to use an LZMA packed kernel image.

Reply:
The bootloader doesn't support LZMA compression.